warehouse/query-core/src/main/java/datawave/query/planner/DefaultQueryPlanner.java

```diff
@@ -1416,11 +1416,11 @@ protected ASTJexlScript timedExpandAnyFieldRegexNodes(QueryStopwatch timers, fin
         } catch (EmptyUnfieldedTermExpansionException e) {
             // The visitor will only throw this if we cannot expand anything resulting in empty query
             NotFoundQueryException qe = new NotFoundQueryException(DatawaveErrorCode.UNFIELDED_QUERY_ZERO_MATCHES, e, MessageFormat.format("Query: ", query));
-            log.info(qe);
+            log.info(String.valueOf(qe));
```

Comment:
I'm gonna stop commenting on these but just do a search for any `String.valueOf` and use the empty message pattern mentioned above.